templates/server.conf.erb

```diff
@@ -33,36 +37,48 @@
 # ca_path:           The ca_certs file contains a set of concatenated “certification authority”
 #                    certificates, which are used to validate certificates passed from the other end
 #                    of the connection.
+# unsafe_autoretry:  If true, retry commands to the database if there is a connection error.
+#                    Warning: if set to true, this setting can result in duplicate records.
+# write_concern:     Write concern of 'majority' or 'all'. When 'all' is specified, 'w' is set to
+#                    number of seeds specified. For version of MongoDB < 2.6, replica_set must also
+#                    be specified. Please note that 'all' will cause Pulp to halt if any of the
+#                    replica set members is not available. 'majority' is used by defau
 
 [database]
-name: <%= scope['pulp::db_name'] %>
+name: <%= scope['pulp::db_name']%>
 seeds: <%= scope['pulp::db_seeds'] %>
-<% if scope['pulp::db_username'] != :undef -%>
+<% if scope['pulp::db_username'] != undef -%>
```

Comment:
i think you left out the ':'  a bunch of these didn't need updates, find/replace error?